Android/wear/src/main/java/dev/saxionroosters/models/Week.java

```diff
@@ -0,0 +1,45 @@
+package dev.saxionroosters.models;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+
+/**
+ * Created by Doppie on 24-2-2016.
+ */
+public class Week implements Serializable {
```

Comment:
Klopt, dit is zeker nog niet mooi. Wou ook nog gaan kijken of ik complete custom models naar de watch kan sturen, ik stuur namelijk nu nog stringarrays.